### ISSUE TITLE
Searching challenge

### ISSUE THREAD

**Issue by 1jamesthompson1** [1jamesthompson1/TAIC-report-summary#163]
## State

There are lots of accident reports published by TAIC and other agencies all around the world.

## Problem

The stated goal of this project is to help the investigators and one of the problems they have is finidng the relevant rpeorts and infromation to a query.

## Solution

A smart database that can be searched using natural language and find the corase grained and fine grain information quickly and intiutavly.

There are two technology ideas that are well suited for this task, which are vector databases #146 and Retrival Augementation and Generation systems #162.

Both of these will be explored and worked on through these tasks:

- [x] #165
- [x] #170

**Comment by 1jamesthompson1** [comment]
After #144 has been put on hold for the time being focus on the searching problem has taken centre stage.

Rather than do a waterfall (which is what the issues are laid out like) it will be completed in a iterative process.

The first step is to get the relevant safety issues, from these it can be feed into the RAG. This will create a basic demo that can be tested and trialed with an investigator from TAIC

**Comment by 1jamesthompson1** [comment]
This seraching problem for now can be considered solved.
The viewer app is doing a suitalbe job of finding the relevant information for the users. The next step is doing useful things with this relevant information.
